docs/Bitbucket.md

```diff
@@ -0,0 +1,107 @@
+| `pipelines.default.variables` | ✅ | Custom variables that are passed to Bitbucket pipeline steps. Each variable defined in a Bitbucket pipeline step is translated to a Buildkite [input step](https://buildkite.com/docs/pipelines/input-step) with/without defaults and allowed values specified below. |
+| `pipelines.default.variables.name` | ✅ | The variables' name: translated to the `key` attribute of a specific `field` of an [input step](https://buildkite.com/docs/pipelines/input-step) (text entry).|
+| `pipelines.default.variables.default` | ✅ | The default variable value if no value is set. Set as the `default` attribute within the `field` of an [input step](https://buildkite.com/docs/pipelines/input-step). |
+| `pipelines.default.variables.description` | ✅ | The description of the variable. Translated to the `text` attribute of a specific `field` within the generated  [input step](https://buildkite.com/docs/pipelines/input-step). |
+| `pipelines.default.variables.allowed-values` | ✅ | The variable's allowed values: each option is translated to a singular `options` object with given value of a specific `field` of an [input step](https://buildkite.com/docs/pipelines/input-step). |
+| `pipelines.pull_request` | ✅ | Application of specific Bitbucket pipeline configuration based for pull requests. Translated to a [step conditional](https://buildkite.com/docs/pipelines/conditionals#conditionals-in-steps) in the corresponding Buildkite pipeline utilising the `pull_request.id`/`BUILDKITE_PULL_REQUEST` and `build.branch`/`BUILDKITE_BRANCH` variables. |
+| `pipelines.pull_request.<branch>` | ✅ | The base branch name/wildcard to apply specific Bitbucket pipeline step configuration within. Apply the configuration for all builds with a `**` wildcard. |
+| `pipelines.pull_request.<branch>.step` | ✅ | Individual step configuration for pull requests builds within a Bitbucket pipeline. See configuration options in this section (`pipelines.default.step.<property>`) for supported/unsupported properties. |
+| `pipelines.tags` | ✅ | Application of specific Bitbucket pipeline configuration based for triggered tag builds. Translated to a [step conditional](https://buildkite.com/docs/pipelines/conditionals#conditionals-in-steps) in the corresponding Buildkite pipeline utilising the `build.tag`/`BUILDKITE_TAG` variable. |
+| `pipelines.tags.<tag>` | ✅ | The tag name/wildcard to apply specific Bitbucket pipeline step configuration within. |
+| `pipelines.tags.<tag>.step` | ✅ | Individual step configuration for triggered tag builds within a Bitbucket pipeline. See configuration options in this section (`pipelines.default.step.<property>`) for supported/unsupported properties. |
```

Comment:
Same as others, I don't think that these two add any kind of useful information (unless you move step configurations to its own section and add a link to it from the step key translation doc)

Reply:
I'll be doing the sub-section for `pipelines` in a second pass - once each one of these suggestions are in place 👍